main/reference/zoe-api/zoe-data-types.md

```diff
@@ -83,11 +54,34 @@ is exclusively yours and no one else has access to it.
 
 ## Keyword
 
-An ASCII identifier string that must begin with an upper case letter
-in order to avoid collisions with JavaScript properties such as `toString`.
+A **Keyword** is a unique identifier string within a contract for tying together the 
+**issuers** in its **proposals**, **payments**, and **payouts**.
+It must be an ASCII-only [identifier](https://developer.mozilla.org/en-US/docs/Glossary/Identifier)
+and start with an upper case letter in order to avoid collisions with JavaScript properties
+such as `toString` when used as a property name in a record.
 (For more detail, see [Why do Zoe keywords have to start with a capital letter? #8241](https://github.com/Agoric/agoric-sdk/discussions/8241).)
 `NaN` and `Infinity` are also not allowed as keywords.
 
+<a id="amountkeywordrecord"></a>
+## KeywordRecord
+
+A **KeywordRecord** is a [CopyRecord](/glossary/#copyrecord) in which every property name
+is a **[Keyword](#keyword)**, such as `harden({ Asset: moolaIssuer, Bid: simoleanIssuer })`.
+Subtypes further constrain property values (for example, an
+**AmountKeywordRecord** is a **KeywordRecord** in which every value is an
+**[Amount](/reference/ertp-api/ertp-data-types.md#amount)** and a
+**PaymentPKeywordRecord** is a **KeywordRecord** in which every value is either a
+**[Payment](/reference/ertp-api/payment.md)** or a Promise for a Payment).
+
+Users submit their **payments** and receive their **payouts** as **KeywordRecords**:
+```js
+quatloosPurse.deposit(payouts.Asset);
```

Comment:
Since `payouts.Asset` may be a promise for a payment, and since `purse.deposit` demands a non-promise for its payment argument, this example is incorrect or at best misleading.